src/main/java/io/cdap/plugin/cdc/common/AvroConverter.java

```diff
@@ -14,10 +14,10 @@
  * the License.
  */
 
-package co.cask.cdc.plugins.common;
+package io.cdap.plugin.cdc.common;
 
-import co.cask.cdap.api.data.format.StructuredRecord;
-import co.cask.cdap.api.data.schema.Schema;
+import io.cdap.cdap.api.data.format.StructuredRecord;
```

Comment:
I believe imports are supposed to be in alphabetical order

Reply:
yes, sure. What is wrong with current order?

Reply:
the io.cdap ones should come after the com.xyz ones

Reply:
I see. Fixed order in all files and checked by maven task.